group_layers/group-layers/README.md

```diff
@@ -27,12 +32,8 @@ The layers in the map will be displayed in a table of contents. Toggle the check
 
 The full extent of a group layer may change when child layers are added/removed. Group layers do not have a spatial reference, but the full extent will have the spatial reference of the first child layer.
 
-Group layers can be saved to web scenes. In web maps, group layers will be flattened in the web map's operational layers.
-
-The implementation shown here makes use of a custom tree cell for displaying layers in a tree view. In the custom
```

Comment:
With the above comment in mind, we should keep this information in here for the Java sample.

Reply:
Yep, I've added this back in. Thanks!